Go 1.16 CI is failing

Even in PRs not modifying Go source, fails with:
```
Error: ../../../../pkg/mod/honnef.co/go/tools@v0.3.1/go/ir/builder.go:36:2: //go:build comment without // +build comment
Error: make: *** [Makefile:111: .install.lint] Error 1
Error: Process completed with exit code 2.
```

--- a/Makefile
+++ b/Makefile
@@ -108,7 +108,7 @@ endif
 install.tools: $(TOOLS:%=.install.%)
 
 .install.lint:
-	go get github.com/golangci/golangci-lint/cmd/golangci-lint@latest
+	go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.45.2
 
 .install.gitvalidation:
 	go get -u github.com/vbatts/git-validation

